reference-architecture/vmware-ansible/README.md

```diff
@@ -19,18 +19,19 @@ rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm*
 yum -y install atomic-openshift-utils \
                   git \
                   pyOpenSSL \
-                  python-simplejson \
                   PyYAML \
-                  python-ldap \
                   python-click \
+                  python-httplib2
                   python-iptools \
+                  python-ldap \
                   python-netaddr \
-                  python-httplib2
-git clone https://github.com/dannvix/pySphere && cd pySphere/ && python setup.py install
+                  python-simplejson \
+                  python-six \
+
 git clone https://github.com/vmware/pyvmomi && cd pyvmomi/ && python setup.py install
 
 # Grabbed the patched vsphere_guest until the PR is merged
-cp vsphere_guest.py /usr/lib/python2.7/site-packages/ansible/modules/core/cloud/vmware/
+cp vmware_guest.py /usr/lib/python2.7/site-packages/ansible/modules/core/cloud/vmware/
```

Comment:
The only problem is I can't call the module as you would in a normal playbook if I dont move it to that directory. 

I could rename it as to not overwrite the existing vmware_guest. vmware_guest_ocp or something like that